web/modules/custom/joinup_invite/src/Form/InviteForm.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace Drupal\joinup_invite\Form;
+
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\CurrentRouteMatch;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\MembershipManager;
+use Drupal\og\OgAccessInterface;
+use Drupal\rdf_entity\UriEncoder;
+use Drupal\user\Entity\User;
+use Drupal\user\UserInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class InviteForm.
+ */
+class InviteForm extends FormBase {
+
+  /**
+   * The current route match service.
+   *
+   * @var \Drupal\Core\Routing\CurrentRouteMatch
+   */
+  protected $currentRouteMatch;
+
+  /**
+   * The entity type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The og membership manager service.
+   *
+   * @var \Drupal\og\MembershipManager
+   */
+  protected $ogMembershipManager;
+
+  /**
+   * The OG access service.
+   *
+   * @var \Drupal\og\OgAccessInterface
+   */
+  protected $ogAccess;
+
+  /**
+   * Constructs a new InviteForm object.
+   *
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $current_route_match
+   *   The current route match service.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
+   *   The entity type manager service.
+   * @param \Drupal\og\MembershipManager $og_membership_manager
+   *   The og membership manager service.
+   * @param \Drupal\og\OgAccessInterface $og_access
+   *   The OG access service.
+   */
+  public function __construct(CurrentRouteMatch $current_route_match, EntityTypeManager $entity_type_manager, MembershipManager $og_membership_manager, OgAccessInterface $og_access) {
+    $this->currentRouteMatch = $current_route_match;
+    $this->entityTypeManager = $entity_type_manager;
+    $this->ogMembershipManager = $og_membership_manager;
+    $this->ogAccess = $og_access;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('current_route_match'),
+      $container->get('entity_type.manager'),
+      $container->get('og.membership_manager'),
+      $container->get('og.access')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'invite_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $entity_type_id = $this->currentRouteMatch->getRouteObject()->getOption('_og_entity_type_id');
```

Comment:
This can be removed and you can update the function declaration with a `RdfInterface $rdf_entity = NULL`.
Extra method parameters will be resolved by `\Drupal\Core\Controller\FormController`.